bin/node/src/chain_spec.rs

```diff
@@ -337,21 +297,14 @@ fn configure_distinct_staking_accounts(
                 .into_iter()
                 .map(|account| (account, TOKEN)),
         )
-        .chain(
-            stashes
-                .clone()
-                .into_iter()
-                .map(|account| (account, MIN_VALIDATOR_BOND + TOKEN)),
-        )
         .collect();
 
     let keys = authorities
         .iter()
-        .zip(stashes.clone())
-        .map(|(auth, stash)| {
+        .map(|auth| {
             (
-                stash.clone(),
-                stash,
+                auth.account_id.clone(),
+                auth.account_id.clone(),
```

Comment:
do we have to clone this two times?

Reply:
We're iterating over shared reference (`iter()`) and later on we're using it in line 316 so we can't pass ownership here